src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -30,31 +31,36 @@ public class BookController {
 
     @Operation(summary = "Get all books", description = "Get a list of all available books")
     @GetMapping
+    @PreAuthorize("hasRole('USER')")
```

Comment:
It wasn't clarified in the task description, but let's allow Admins to do user roles too, for that, you need to remove @PreAuthorize("hasRole('USER')") and leave only @PreAuthorize("hasRole('ADMIN')") - others are secured in SecurityConfig